### ISSUE TITLE
How to send formated text & picture as response to inline bot

### ISSUE THREAD

**Issue by pasondag** [pengrad/java-telegram-bot-api#43]
Hi @pengrad ,

I see that there are several type of content that we can send to the picker

example picker with text & photos: 
`new InlineQueryResultArticle("id", "title", "response send to user if selecting").thumbUrl("url");`

However, how do I format the message send to the user after the user made a selection in the picker? As example, suppose the user select the content above, how can I send back a message with a picture with text, where some part of the text is in bold and some words are links?

I have tried using markdown and html, but none are interpreted. I guess there must be an option to enable HTML usage.

Basically something similar as what the popcornbot (https://telegram.me/imdb) does.

Thanks in advance.
PA

**Comment by pengrad** [comment]
Hi,
`SendMessage` with `parse_mode` is what you need. 
Picture from imdb bot is fetched by telegram. 
But seems telegram by default also fetch some text, not only picture. 
Maybe `disable_web_page_preview` will help...

**Comment by pasondag** [comment]
Thanks for the answer @pengrad . I don't see how to use `SendMessage` in inline bot context. Say you have the following:

```java
if (update.inlineQuery() != null) {

        InlineQueryResult r1 = new InlineQueryResultArticle("1", "title 1", "content 1");
        InlineQueryResult r2 = new InlineQueryResultArticle("2", "title 2", "content 2");

        BaseResponse response =
                bot.execute(
                        new AnswerInlineQuery(
                                update.inlineQuery().id(), r1, r2).cacheTime(0));
    }
```

How do we use `SendMessage` to make content 1 or content 2 be html? 

Similarly I am not sure were to input `disable_web_page_preview`

**Comment by pengrad** [comment]
Ah, understand, you need this in inline mode...
Try to play with `parse_mode` in `input_message_content`.
 Its a parameter of InlineQueryResultArticle

**Comment by pasondag** [comment]
Yeah that was it. Thanks @pengrad.

For anyone having the same issue in the future here is how to do html on inline response:
```java
new InlineQueryResultArticle(
         "1", 
         title, 
         new InputTextMessageContent("<b>html content</b>").parseMode(HTML));
```

**Comment by patrickbard** [comment]
> **For anyone having the same issue in the future** here is how to do html on inline response:

Greatings from the future :+1:
